lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+#: messages.c:109
+msgid "Is steel sheet on heatbed?"
+msgstr "Ligger metallskiva pa v\xe1rmeb\xe1dden?"
+
+# MSG_LAST_PRINT_FAILURES c=20
+#: messages.c:57
+msgid "Last print failures"
+msgstr "Senaste utskriftsfel"
+
+# MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
+#: messages.c:122
+msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
+msgstr "Hej, jag \xe1r din Prusa i3-skrivare. Jag guidar dig genom en kort installationsprocess, d\xe1r Z-axeln kommer att kalibreras. Sedan \xe1r du redo att skriva ut."
+
+# MSG_ADDITIONAL_SHEETS c=20 r=9
+#: ultralcd.cpp:5029
+msgid "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets."
+msgstr "Om du har ytterligare metallskivor, kalibrera deras f\xefrinst\xe1llningar i Inst\xe1llningar - HW inst\xe1llning - Metallskiva."
+
+# MSG_LAST_PRINT c=18
+#: messages.c:56
+msgid "Last print"
+msgstr "Senaste utskrift"
+
+# MSG_SELFTEST_EXTRUDER_FAN c=20
+#: messages.c:89
+msgid "Left hotend fan?"
+msgstr "V\xe1nst hotend fl\xe1kt?"
+
+# MSG_LEFT c=10
+#: ultralcd.cpp:2844
+msgid "Left"
+msgstr "V\xe1nster"
+
+# MSG_BED_CORRECTION_LEFT c=14
+#: ultralcd.cpp:3114
+msgid "Left side [um]"
+msgstr "V\xe1nstsida [um]"
+
+# MSG_LIN_CORRECTION c=18
+#: ultralcd.cpp:5702
+msgid "Lin. correction"
+msgstr "Linj\xe1r korrektion"
+
+# MSG_BABYSTEP_Z c=18
+#: messages.c:12
+msgid "Live adjust Z"
+msgstr "Live justera Z"
+
+# MSG_INSERT_FIL c=20 r=6
+#: ultralcd.cpp:7380
+msgid "Insert the filament (do not load it) into the extruder and then press the knob."
+msgstr "Infoga filamentet (ladda inte in det) i extrudern och tryck sedan pa knappen."
+
+# MSG_LOAD_FILAMENT c=17
+#: messages.c:58
+msgid "Load filament"
+msgstr "Ladda filament"
+
+# MSG_LOADING_COLOR c=20
+#: ultralcd.cpp:2516
+msgid "Loading color"
+msgstr "Laddar f\xe1rg"
+
+# MSG_LOADING_FILAMENT c=20
+#: messages.c:59
+msgid "Loading filament"
+msgstr "Laddar filament"
+
+# MSG_ITERATION c=12
+#: messages.c:53
+msgid "Iteration"
+msgstr "Iteration"
+
+# MSG_LOOSE_PULLEY c=20
+#: ultralcd.cpp:8167
+msgid "Loose pulley"
+msgstr "Lossa pulley"
+
+# MSG_LOAD_TO_NOZZLE c=18
+#: ultralcd.cpp:6717
+msgid "Load to nozzle"
+msgstr "Ladd till munstyck"
+
+# MSG_M117_V2_CALIBRATION c=25
+#: messages.c:62
+msgid "M117 First layer cal."
+msgstr "M117 F\xefrst lager kalibr."
+
+# MSG_MAIN c=18
+#: messages.c:63
+msgid "Main"
+msgstr "Huvudsaklig"
+
+# MSG_BL_HIGH c=12
+#: messages.c:155
+msgid "Level Bright"
+msgstr "Ljusniva"
+
+# MSG_BL_LOW c=12
+#: messages.c:156
+msgid "Level Dimmed"
+msgstr "Niva d\xe1mpad"
+
+# MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=20 r=3
+#: messages.c:67
+msgid "Measuring reference height of calibration point"
+msgstr "M\xe1tning av referensh\xefjd f\xefr kalibreringspunkt"
+
+# MSG_MESH_BED_LEVELING c=18
+#: messages.c:148
+msgid "Mesh Bed Leveling"
+msgstr "B\xe1dd nivautj\xe1mning"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.